content/zh/docs/devops-user-guide/examples/create-multi-cluster-pipeline.md

```diff
@@ -0,0 +1,263 @@
+
+### Step 5: Create credentials
+
+1. Log out of the console and log back in as `project-regular`. On the **DevOps Projects** page, click the DevOps project **multicluster-demo**.
+
+2. On the **DevOps Credentials** page, you need to create the credentials as shown in the table below. For more information about how to create credentials, refer to [Credential Management](../../how-to-use/credential-management/#create-credentials) and [Create a Pipeline Using a Jenkinsfile](../../how-to-use/create-a-pipeline-using-jenkinsfile/#step-1-create-credentials).
+
+   | Credential ID | Type                | Where to Use                       |
+   | ------------- | ------------------- | ---------------------------------- |
+   | host          | kubeconfig          | The Host Cluster for testing       |
+   | shire         | kubeconfig          | The Member Cluster for production  |
+   | rohan         | kubeconfig          | The Member Cluster for development |
+   | dockerhub-id  | Account Credentials | Docker Hub                         |
+   | sonar-token   | Secret Text         | SonarQube                          |
+
+   {{< notice note >}}
+
+   You have to manually enter the kubeconfig of your Member Clusters when creating the kubeconfig credentials `shire` and `rohan`. Make sure your Host Cluster can access the APIServer addresses of your Member Clusters.
+
+   {{</ notice >}}
+
+3. You will have five credentials in total.
+
+   ![credentials-created](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/credentials-created.png)
+
+### Step 6: Create a pipeline
+
+1. Go to the **Pipelines** page and click **Create**. In the dialog that appears, enter `build-and-deploy-application` for **Name** and click **Next**.
+
+   ![pipeline-name](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/pipeline-name.png)
+
+2. In the **Advanced Settings** tab, click **Create** to use the default settings.
+
+3. The pipeline created will display in the list. Click it to go to its detail page.
+
+   ![pipeline-created](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/pipeline-created.png)
+
+4. Click **Edit Jenkinsfile** and copy and paste the following contents. Make sure you replace the value of `DOCKERHUB_NAMESPACE` with your own value, and then click **OK**.
+
+   ```groovy
+   pipeline {
+     agent {
+       node {
+         label 'maven'
+       }
+   
+     }
+     parameters {
+           string(name:'BRANCH_NAME',defaultValue: 'master',description:'')
+       }
+     environment {
+           DOCKER_CREDENTIAL_ID = 'dockerhub-id'
+           PROD_KUBECONFIG_CREDENTIAL_ID = 'shire'
+           TEST_KUBECONFIG_CREDENTIAL_ID = 'host'
+           DEV_KUBECONFIG_CREDENTIAL_ID = 'rohan'
+   
+           REGISTRY = 'docker.io'
+           DOCKERHUB_NAMESPACE = 'shaowenchen'
+           APP_NAME = 'devops-java-sample'
+           SONAR_CREDENTIAL_ID = 'sonar-token'
```

Comment:
In this tutorial, the Pipeline relies on the Sonarqube. But I didn't find any description or links about how to integrate Sonarqube with KubeSphere DevOps.

Reply:
Good catch. I put the information in a note. I will move it to `Prerequisites`.